recipe-server/client/control/components/Fields.js

```diff
@@ -26,23 +26,57 @@ export function buildControlField({
   children,
   ...args // eslint-disable-line comma-dangle
 }) {
+  const WrappingElement = label ? 'label' : 'div';
   return (
-    <label className={`${className} form-field`}>
+    <WrappingElement className={`${className} form-field`}>
       <span className="label">{label}</span>
       {error && <span className="error">{error}</span>}
-      <InputComponent {...input} {...args}>
+      <InputComponent {...input} {...args} input={input}>
```

Comment:
The `CheckboxGroup` code I was using had pulled values from `input` and for some reason I thought it had to be passed as a whole object. I've removed this extra prop.